Feature/Updated validation to allow for missing sigma (#503)
PowerGridModel/power-grid-model: docs/user_manual/components.md

| ------------------------ | ----------------------------------------------------------------------------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :---------------------------------: | :------: | :--------------------------------------------------: |
| `measured_terminal_type` | {py:class}`MeasuredTerminalType <power_grid_model.enum.MeasuredTerminalType>` | -                | indicate if it measures an `appliance` or a `branch`                                                                                                                                        |              &#10004;               | &#10060; | the terminal type should match the `measured_object` |

> | `power_sigma`            | `double`                                                                      | volt-ampere (VA) | standard deviation of the measurement error. Usually this is the absolute measurement error range divided by 3. See {hoverxreftooltip}`user_manual/components:Power Sensor Concrete Types`. | &#10024; only for state estimation. See the explanation below. | &#10004; |                        `> 0`                         |

Reviewer: we also have to think about the possiblity that in the future we provide e.g. only a single-phase asym power sensor or maybe even only an active power sensor or only a reactive power sensor. is just the comment `See the explanation below.` sufficient? maybe make this something like

```suggestion
| `power_sigma`            | `double`                                                                      | volt-ampere (VA) | standard deviation of the measurement error. Usually this is the absolute measurement error range divided by 3. See {hoverxreftooltip}`user_manual/components:Power Sensor Concrete Types`. | &#10024; in certain cases for state estimation. See the explanation below. | &#10004; |                        `> 0`                         |
```

Author: Changes included in the latest commit.

> | `power_sigma`            | `double`                                                                      | volt-ampere (VA) | standard deviation of the measurement error. Usually this is the absolute measurement error range divided by 3. See {hoverxreftooltip}`user_manual/components:Power Sensor Concrete Types`. | &#10024; in certain cases for state estimation. See the explanation below. | &#10004; |                        `> 0`                         |

Reviewer: now that i read this back, it's in a different (sub)section, so maybe this makes it more clear:

```suggestion
| `power_sigma`            | `double`                                                                      | volt-ampere (VA) | standard deviation of the measurement error. Usually this is the absolute measurement error range divided by 3. See {hoverxreftooltip}`user_manual/components:Power Sensor Concrete Types`. | &#10024; in certain cases for state estimation. See the explanation for [concrete types](#power-sensor-concrete-types) below. | &#10004; |                        `> 0`                         |
```

| `p_sigma`    | `RealValueInput` | watt (W)                   | standard deviation of the active power measurement error. Usually this is the absolute measurement error range divided by 3.   | &#10060; see the explanation below. | &#10004; | `> 0` |
| `q_sigma`    | `RealValueInput` | volt-ampere-reactive (var) | standard deviation of the reactive power measurement error. Usually this is the absolute measurement error range divided by 3. | &#10060; see the explanation below. | &#10004; | `> 0` |

Valid combinations of `power_sigma`, `p_sigma` and `q_sigma` are:

> | `power_sigma` | `p_sigma` | `q_sigma` |  result  |

Reviewer: ```suggestion

| `power_sigma` | `p_sigma` | `q_sigma` |  result  |
```

> | `power_sigma` | `p_sigma` | `q_sigma` |  result  |

Reviewer: nitpick
```suggestion
| `power_sigma` | `p_sigma` | `q_sigma` |  result  |

```

Reviewer: Whoops sorry no an empty line before the table begins, apologies

|:-------------:|:---------:|:---------:|:--------:|
|    &#10004;   |  &#10004; |  &#10004; |  valid   |
|    &#10004;   |  &#10004; | &#10060;  | invalid  |
|    &#10004;   | &#10060;  |  &#10004; | invalid  |
|    &#10004;   | &#10060;  | &#10060;  |  valid   |
|   &#10060;    |  &#10004; |  &#10004; |  valid   |
|   &#10060;    |  &#10004; | &#10060;  | invalid  |
|   &#10060;    | &#10060;  |  &#10004; | invalid  |

> |   &#10060;    | &#10060;  | &#10060;  | invalid  |

Reviewer: maybe instead provide just a `x`-mark for "provided" and then a &#10004; or a &#10060; for the `valid/invalid` column. i think that looks easier

Author: Changes included in the latest commit.
